indispensable-asn1/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Encoding.kt

```diff
@@ -0,0 +1,59 @@
+package at.asitplus.signum.indispensable.asn1
+
+import kotlinx.io.*
+import kotlinx.io.unsafe.UnsafeBufferOperations
+
+@OptIn(UnsafeIoApi::class)
+fun ByteArray.wrapInUnsafeSource(): Source = Buffer().apply {
```

Comment:
`internal`

Reply:
I'd rather expose it. could be handy for custom asn.1 foo

Reply:
then give it a `RequireOptIn`, don't just leave the footgun lying around (yes, even if it says `Unsafe`)